fixtures/fixtures.ts

```diff
@@ -0,0 +1,49 @@
+import { test as base } from "@playwright/test";
+import { HomePage } from "../pages/home.page";
+import { LoginPage } from "../pages/login.page";
+import { ProductPage } from "../pages/product.page";
+import { CartPage } from "../pages/cart.page";
+import { CheckoutPage } from "../pages/checkout.page";
+
+
+type MyFixtures = {
+  homePage: HomePage;
+  loginPage: LoginPage;
+  productPage: ProductPage;
+  loggedInPage: HomePage;
+  cartPage: CartPage;
+  checkoutPage: CheckoutPage;
+};
+
+export const test = base.extend<MyFixtures>({
+  homePage: async ({ page }, use) => {
+    const homePage = new HomePage(page);
+    await use(homePage);
+  },
+  loginPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
+    await use(loginPage);
+  },
+  productPage: async ({ page }, use) => {
+    const productPage = new ProductPage(page);
+    await use(productPage);
+  },
+  loggedInPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
+    await page.goto(process.env.WEB_URL + '/auth/login');
```

Comment:
```suggestion
    await page.goto('/auth/login');
```
Should work without specifying the exact URL